python/tox.ini

```diff
@@ -65,6 +69,9 @@ commands =
 [testenv:type-check]
 deps =
     mypy
+    types-pytz
```

Comment:
You could also install these types implicitly: https://mypy.readthedocs.io/en/stable/command_line.html#miscellaneous
This has to be combined with the non-interactive flag: https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-non-interactive